_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -187,6 +187,9 @@ function page_admin_scripts() {
 		// Add objects to be passed to the initial state of the app.
 		// Use wp_add_inline_script instead of wp_localize_script, see https://core.trac.wordpress.org/ticket/25280.
 		wp_add_inline_script( 'react-plugin', 'var Initial_State=JSON.parse(decodeURIComponent("' . rawurlencode( wp_json_encode( $this->get_initial_state() ) ) . '"));', 'before' );
+
+		// This will set the default URL of the jp_redirects lib.
+		wp_add_inline_script( 'react-plugin', 'var jetpack_redirects = { currentSiteRawUrl: "' . Jetpack::build_raw_urls( get_home_url() ) . '" };', 'before' );
```

Comment:
In this case we could use the existing variable under InitialState, but I decided to make its own variable to move in the direction of having this library replaced by a npm package -> https://github.com/Automattic/wp-calypso/pull/41497

This will also be used, for example, for Blocks, and we won't have initialState there.